flake.nix

```diff
@@ -1033,17 +1049,41 @@
                 name = "kusama-local-dali-dev-karura-dev.json";
                 text = "${builtins.toJSON config}";
               };
+            in writeShellApplication {
+              name = "run-rococo-dali-karura";
+              text = ''
+                cat ${config-file}
+                rm -rf /tmp/polkadot-launch
+                ${packages.polkadot-launch}/bin/polkadot-launch ${config-file} --verbose
+              '';
+            };
+
+            devnet-rococo-picasso-karura-statemine = let
```

Comment:
devnet-all? or devnet-release or devnet-persistent? assuming list will grow with other things . so it can just devnet-<name where it publushed>